packages/compass-web/src/entrypoint.tsx

```diff
@@ -268,11 +269,14 @@ const CompassWeb = ({
       enableAggregationBuilderRunPipeline: true,
       enableAggregationBuilderExtraOptions: true,
       enableImportExport: false,
-      enableGenAIFeatures: false,
+      enableGenAIFeatures: true,
+      enableGenAIFeaturesAtlasProject: false,
+      enableGenAISampleDocumentPassingOnAtlasProject: false,
+      enableGenAIFeaturesAtlasOrg: false,
       enableMultipleConnectionSystem: true,
       enablePerformanceAdvisorBanner: true,
       cloudFeatureRolloutAccess: {
-        GEN_AI_COMPASS: false,
+        GEN_AI_COMPASS: true,
```

Comment:
We'll want to set this to `false` before we merge since this value is retrieved from the `hello` endpoint request we make when the AtlasAiService is initialized.

Reply:
fixed!